tensorflow_probability/python/mcmc/transformed_kernel.py

```diff
@@ -110,6 +106,33 @@ def transformed_log_prob_fn(*state_parts):
   return transformed_log_prob_fn
 
 
+def target_log_prob_getter(kernel):
```

Comment:
Generally it's a good idea to name functions so that the call site reads as either a noun phrase describing the result computed on those arguments, or as a verb phrase describing the action performed on those arguments.  So, `get_target_log_prob(my_kernel)` parses much more cleanly than `target_log_prob_getter(my_kernel)`.  This is because the latter requires the reader to mentally form the image of a `target_log_prob_getter`, infer what that does, and infer what the result will be in the setting of `my_kernel`; whereas the former has one fewer mental step.

Given the fairly strong conventional meaning of `get`, `set`, and `update`, I think you should rename this to `get_target_log_prob`, and the next one to `update_target_log_prob`.

By the way, the conventional difference between `set` and `update` is that `set` implies mutating the object to change a value, whereas `update` implies constructing a new object that is identical except for the changed value.  I also suggest below that you change the setter to do the latter.

Reply:
Done.